tutorials-dp0-3/portal-dp0-3-3.rst

```diff
@@ -0,0 +1,421 @@
+better illustrate how the object moves across the sky.
+In the plot panel, click on the "Settings" icon (double gears) to open the "Plot Parameters"
+pop-up window.
+Under "Trace Options", for "Color Map" enter "midPointMjdTai" and for "Color Scale" enter "Rainbow".
+Then click "Apply".
+
+.. figure:: /_static/portal_tut03_step03c.png
+    :width: 600
+    :name: portal_tut03_step03c
+    :alt: A screenshot of the plot of sky coordinates colored as a function of time.
+
+    The 10 loops in the object's path on the sky is a result of Earth's orbital period and the 10-year LSST duration.  
+    Purple color corresponds to earlier observtations, and the red color corresponds to the later observations.  
+
+
+3.6. Clear the query and results and return to the RSP TAP Search form.
+
+.. _DP0-3-Portal-3-Step-4:
+
+Step 4. Plot the time-domain quantities for the TNO
+===================================================
+
+**Note** that no time domain evolution in object brightness was included in the DP0.3 simulation
+(e.g., rotation curves for non-spherical objects, outgassing events).
+All changes in the brightness of DP0.3 objects with time are due to changes in the distance and phase angle from Earth.  
+
+4.1. Execute the following ADQL query to retrieve the r-band magnitudes, phase angles,
+heliocentric and topocentric distances, and time of the observations for the TNO explored in Step 3.
+
+.. code-block:: SQL 
+
+    SELECT ds.midPointMjdTai, ds.mag, ds.band, 
+    ss.phaseAngle, ss.topocentricDist, ss.heliocentricDist 
+    FROM dp03_catalogs_10yr.DiaSource AS ds 
+    JOIN dp03_catalogs_10yr.SSSource AS ss ON ds.diaSourceId = ss.diaSourceId
+    WHERE ss.ssObjectId = -735085100561880491
+    AND ds.band = 'r'
+
+4.2. The default plot will have the r-band magnitude as a function of time.  
+Use the plot "Settings" function to add new scatter plots showing the magnitude and the phase angle as a function of time, ``midPointMjdTai - 60000``.  
+This will result in the left two plots, as on the screenshot below.   Note that these quantities are not correlated with time.
+Add a new scatter plot showing the r-band magnitude as a function of phase angle (right plot), showing that the phase angle and r-band magnitude are correlated.
+
+.. figure:: /_static/portal_tut03_step04a.png
+    :name: portal_tut03_step04a
+    :width: 600
+    :alt: A screenshot of three plots showing magnitude and phase angle are not correlated with time, and that magnitude is correlated with phase angle.
+
+    Three plots demonstrating that magnitude and phase angle are correlated with each other, but not with time.
+
+4.3.  Plot the topocentric and heliocentric distances of the object as a function of time.  
+Here, use the columns in the table which you generated in Step 4.1.  
+First, delete two of the the three plots prepared in Step 4.2 by clicking on the blue ``X`` in the upper right-hand part of the plot panels to make space for new plots.  
```

Comment:
The use of double quotes `` is inappropriate for X here because in this case X is not a variable or column name.

Reply:
ok